rosdep/python.yaml

```diff
@@ -6195,6 +6195,13 @@ python3-decorator:
   osx:
     pip: [decorator]
   ubuntu: [python3-decorator]
+python3-deepdiff-pip:
```

Comment:
So it looks like this actually has a system package as of Debian Buster: https://packages.debian.org/search?keywords=deepdiff&searchon=names&suite=all&section=all .  It also looks like it has a package in Ubuntu since 20.04: https://packages.ubuntu.com/search?keywords=deepdiff&searchon=names&suite=all&section=all .  I didn't check Fedora, but I think we should switch to using the system-packaged versions instead.

Reply:
Ah, indeed! Commit updated, as well as the OP description copied below for email notification purpose but will be deleted after the post).

----

## Package name:

deepdiff

## Package Upstream Source:

https://pypi.org/project/deepdiff/

Py2 version is already added in https://github.com/ros/rosdistro/pull/15606

## Purpose of using this:

- Help Python application to deal with dict, iterables

# Distro packaging links:

- Debian: https://packages.debian.org/
  - REQUIRED
  - https://packages.debian.org/search?keywords=%20deepdiff
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
   - https://packages.ubuntu.com/search?keywords=deepdiff&searchon=names&suite=all&section=all
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
  - https://src.fedoraproject.org/rpms/python-deepdiff
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
  - https://archlinux.org/packages/community/any/python-deepdiff/
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
  - Not found
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
  - Not found